modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -33,6 +48,17 @@ def show
       render_higher_level_review_not_found(params[:id])
     end
 
+    def validate
```

Comment:
In Decision Reviews, the body for these `validate` actions was housed in a second, dedicated method that was called inside `validate` - there is no logic here, so I removed that second function for readability's sake